llvmlite/binding/__init__.py

```diff
@@ -7,6 +7,7 @@
 from .linker import *
 from .module import *
 from .options import *
+from .newpassmangers import *
```

Comment:
This is mis-spelled:

```suggestion
from .newpassmanagers import *
```

Reply:
Thanks for catching this, seems like I mis-spelled in the file name. I don't want to ruin the review history of the file so will rename it when the pr is merge ready. Let me know if you suggest otherwise :)

Reply:
Thanks - I think the old review comments will still be visible in the "conversation" tab, and I think I'm pretty happy with the resolution of the old comments, so feel free to rename it as soon as is convenient :slightly_smiling_face: